src/shared/sam/debugger/pythonSamDebug.ts

```diff
@@ -153,46 +150,9 @@ export async function invokePythonLambda(
 }
 
 export async function waitForPythonDebugAdapter(debugPort: number, timeout: Timeout, channelLogger: ChannelLogger) {
```

Comment:
Clean up the constructor, the change removes the need for all of these.

Reply:
The signature matches `onWillAttachDebugger`